qtaf-allure-plugin/pom.xml

```diff
@@ -20,7 +20,7 @@
         <dependency>
             <groupId>de.qytera</groupId>
             <artifactId>qtaf-core</artifactId>
-            <version>0.0.9</version>
+            <version>LOCAL_2023_04_27_001</version>
```

Comment:
Please don't push `pom` files containing development versions.

Reply:
Updated version to 0.1.0